resources/definitions/tenlog_printer.def.json

```
@@ -0,0 +1,295 @@
+            "ultimaker_cpe_transparent",
+            "ultimaker_cpe_white",
+            "ultimaker_cpe_yellow",
+            "ultimaker_nylon_black",
+            "ultimaker_nylon_transparent",
+            "ultimaker_pc_black",
+            "ultimaker_pc_transparent",
+            "ultimaker_pc_white",
+            "ultimaker_pla_black",
+            "ultimaker_pla_blue",
+            "ultimaker_pla_green",
+            "ultimaker_pla_magenta",
+            "ultimaker_pla_orange",
+            "ultimaker_pla_pearl-white",
+            "ultimaker_pla_red",
+            "ultimaker_pla_silver-metallic",
+            "ultimaker_pla_transparent",
+            "ultimaker_pla_white",
+            "ultimaker_pla_yellow",
+            "ultimaker_pp_transparent",
+            "ultimaker_pva",
+            "ultimaker_tough_pla_black",
+            "ultimaker_tough_pla_green",
+            "ultimaker_tough_pla_red",
+            "ultimaker_tough_pla_white",
+            "ultimaker_tpu_black",
+            "ultimaker_tpu_blue",
+            "ultimaker_tpu_red",
+            "ultimaker_tpu_white",
+            "verbatim_bvoh_175",
+            "zyyx_pro_flex",
+            "zyyx_pro_pla"
+        ]
+    },
+    "overrides": {
+        "machine_name": { "default_value": "Tenlog Printer" },
+        "machine_start_gcode": {
+            "default_value": "G21 ;metric values\nG90 ;absolute positioning\nM82 ;set extruder to absolute mode\nM107 ;start with the fan off\nG28 ;Move to min endstops\nG92 E0 ;Reset Extruder\nG1 E15 F1500\nG1 Z15.0 F3000 ;move the header up 15mm\nM117 Printing..."
+        },
+        "machine_end_gcode": {
+            "default_value": "M104 T0 S0 ;1st extruder heater off\nM104 T1 S0 ;2nd extruder heater off\nM140 S0 ;heated bed heater off (if you have it)\nG91 ;relative positioning\nG1 E-1 F300  ;retract the filament a bit before lifting the nozzle, to release some of the pressure\nG1 Z+0.5 E-5 X-20 Y-20 F9000 ;move Z up a bit and retract filament even more\nG28 X0 Y0 ;move X/Y to min endstops, so the head is out of the way\nM84 ;steppers off\nG90 ;absolute positioning"
+        },
+
+        "machine_extruder_count": { "default_value": 2 },
+
+        "machine_max_feedrate_x": { "value": 500 },
+        "machine_max_feedrate_y": { "value": 500 },
+        "machine_max_feedrate_z": { "value": 10 },
+        "machine_max_feedrate_e": { "value": 80 },
+
+        "machine_max_acceleration_x": { "value": 500 },
+        "machine_max_acceleration_y": { "value": 500 },
+        "machine_max_acceleration_z": { "value": 100 },
+        "machine_max_acceleration_e": { "value": 5000 },
+        "machine_acceleration": { "value": 500 },
+
+        "layer_height_0":{ "default_value": 0.2	},
+        "initial_layer_line_width_factor":{"default_value": 130.0},
+
+        "skin_overlap": { "value": 25 },
+	"zig_zaggify_infill":{ "value": true},
+	"infill_before_walls":{"value": true},
+
+	"machine_max_jerk_xy": { "value": 10 },
```

Mix of tabs and spaces.

```suggestion
        "zig_zaggify_infill":{ "value": true},
        "infill_before_walls":{"value": true},

        "machine_max_jerk_xy": { "value": 10 },
```